### PR TITLE
[Michelle] added "Announcing Junior Track" blog post

### PR DESCRIPTION
**Pull request by michelleerica**

### CODE REVIEW

File: src/content/posts/accouncing-juniortrack.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Buy your early bird tickets now - only $50: http://next.dddsydney.com.au/tickets/
+
+**Benefits of attending DDD Sydney:**
+
+- discover something new in the dev world
+- meet like-minded individuals, including other juniors!
+- explore dev, design and data
+- learn skills to further your career
+Find out more at http://next.dddsydney.com.au/
```
Comment (aaronpowell) [inline src/content/posts/accouncing-juniortrack.md]: Need a new line between 24 and 25, else they'll merge together

File: src/content/posts/accouncing-juniortrack.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- Dev-ops
+- Creative Coding
+- IoT
+- Mobile
+- Career development
+- Your journey
+- UX, UI, Design
+- Data
+
+Don't forget, we're holding a free CFP workshop Sunday 20 May to help you brainstorm and refine your submission. RSVP [here] (https://www.eventbrite.com.au/e/ddd-sydney-call-for-papers-workshop-tickets-45190507023)
```
Comment (aaronpowell) [inline src/content/posts/accouncing-juniortrack.md]: Need to strip the space between the `]` and `(` so Markdown will turn it into a proper link.

**Comment by aaronpowell** [comment]
I'll schedule this to publish at 9am tomorrow and then schedule a tweet/LinkedIn post at midday, so can you put this for the `date` field:

`2018-05-011T09:00:00+10:00`

**Comment by michelleerica** [comment]
Thanks, Aaron! I've updated the post.

**Pull request merged**